ci/cli_merge_sign.sh

```diff
@@ -48,7 +48,13 @@ done
 # Create a universal binary.
 lipo -create -output cli_darwin_universal cli_darwin_arm64 cli_darwin_amd64
 
-gon ci/gon.hcl
+export TEAM_ID="SZCNTABEXY"
+export APP_IDENTITY="Developer ID Application: Pixie Labs Inc. (${TEAM_ID})"
+codesign -f -v --timestamp --options runtime -s "${APP_IDENTITY}" cli_darwin_amd64 cli_darwin_arm64 cli_darwin_universal
```

Comment:
This was modeled off how gon runs codesign ([source](https://github.com/mitchellh/gon/blob/2d4f161ccecd1aae878f4416d5ccd622b1b01fdb/sign/sign.go#L66-L73))